<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Error Message is Displayed]]>
    </line>
    <line>
      <![CDATA[Error Message is Displayed]]>
    </line>
    <line>
      <![CDATA[Error Message is not displated ]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="31544" started-at="2017-03-30T19:28:43Z" finished-at="2017-03-30T19:29:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="31544" started-at="2017-03-30T19:28:43Z" finished-at="2017-03-30T19:29:14Z">
      <class name="Script.VerifyInvalidLogin">
        <test-method status="PASS" signature="precondition()[pri:0, instance:Script.VerifyInvalidLogin@47c62251]" name="precondition" is-config="true" duration-ms="6049" started-at="2017-03-31T00:58:43Z" finished-at="2017-03-31T00:58:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="testVerifyInvalidLogin()[pri:0, instance:Script.VerifyInvalidLogin@47c62251]" name="testVerifyInvalidLogin" duration-ms="23601" started-at="2017-03-31T00:58:49Z" finished-at="2017-03-31T00:59:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at Script.VerifyInvalidLogin.testVerifyInvalidLogin(VerifyInvalidLogin.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Error Message is Displayed]]>
            </line>
            <line>
              <![CDATA[Error Message is Displayed]]>
            </line>
            <line>
              <![CDATA[Error Message is not displated ]]>
            </line>
          </reporter-output>
        </test-method> <!-- testVerifyInvalidLogin -->
        <test-method status="PASS" signature="postcondition()[pri:0, instance:Script.VerifyInvalidLogin@47c62251]" name="postcondition" is-config="true" duration-ms="1876" started-at="2017-03-31T00:59:13Z" finished-at="2017-03-31T00:59:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postcondition -->
      </class> <!-- Script.VerifyInvalidLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
